src/main/java/org/embulk/input/marketo/delegate/LeadWithProgramInputPlugin.java

```diff
@@ -18,14 +18,15 @@
  */
 public class LeadWithProgramInputPlugin extends MarketoBaseInputPluginDelegate<LeadWithProgramInputPlugin.PluginTask>
 {
-    public interface PluginTask extends MarketoBaseInputPluginDelegate.PluginTask
+    public interface PluginTask extends MarketoBaseInputPluginDelegate.PluginTask, LeadServiceResponseMapperBuilder.PluginTask
     {
     }
 
     @Override
     protected Iterator<ServiceRecord> getServiceRecords(MarketoService marketoService, PluginTask task)
     {
         List<String> fieldNames = task.getExtractedFields();
+        fieldNames.remove(MarketoUtils.PROGRAM_ID_COLUMN_NAME);
```

Comment:
Is there something wrong with this field? If there is, could you add a comment to explain?